Package.pins

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why do need a Package.pins? Do we want to pin dependencies to a release? I think that is the point of this file

Reply:
Answered on slack

Reply:
It's our standard "Don't pin package versions" Package.pins, and is the same file that was present in the latest (1.8.4) tag.  An easier way to evaluate this PR might be to see what the net result is when compared to that tag:
https://github.com/IBM-Swift/Kitura-StencilTemplateEngine/compare/1.8.4...issue.swift311